storybook/src/components/TextArea/TextArea.stories.tsx

```diff
@@ -24,6 +24,9 @@ const meta = {
       },
       description: 'The width, expressed in the average number of characters.',
     },
+    defaultValue: {
+      table: { disable: true },
+    },
```

Comment:
This field doesn’t get hidden for `TextInput`. I agree that hiding is the correct choice because editing the field does nothing to the content. I suggest we make this a global setting as we do for `children` in `storybook/config/preview.tsx`.

Reply:
I've hidden `defaultValue` for Text Input as well.

I'm not sure we should make this a global setting. This is only relevant for inputs where you can input text, which currently is only TextInput, TextArea and PasswordInput. It'll probably be more confusing if we hide that in a global setting. 

Should we show the `defaultValue` control for the `disabled` stories? Currently, there is no way to change the text there.

Reply:
> I'm not sure we should make this a global setting. This is only relevant for inputs where you can input text, which currently is only TextInput, TextArea and PasswordInput. It'll probably be more confusing if we hide that in a global setting.

Fair enough.

> Should we show the defaultValue control for the disabled stories? Currently, there is no way to change the text there.

People can switch the `disabled` prop off and on again. I would say that’s easy enough to prevent having to add this exception.